app/javascript/src/Policies/components/PoliciesForm.jsx

```diff
@@ -32,7 +32,7 @@ function PoliciesForm ({
   closePolicyConfig}: Props) {
   const onSubmit = (policy) => {
     return ({formData, schema}) => {
-      submitForm({...policy, ...{schema, configuration: formData}})
+      submitForm({...policy, ...{configuration: schema, data: formData}})
```

Comment:
Here there's something off. What's _schema_ and what's _configuration_? I think there should be only one of those so probably just a naming issue.

Reply:
It's confusing because what we are using in the Registry and Chain is not the same way we are using it in Custom Policies. The idea is to unify the most of it after this PR.